javafx7-mapview/src/main/java/slash/navigation/mapview/browser/JavaFX7WebViewMapView.java

```diff
@@ -84,6 +86,12 @@ protected WebView getWebView() {
     private WebView createWebView() {
         try {
             final WebView webView = new WebView();
+
+            final int scale = preferences.getInt(BROWSER_SCALE_FACTOR_PREFERENCE, 100);
+            if (scale != 100) {
+                setBrowserScale(webView, scale/100.);
+            }
```

Comment:
Da haben sich unsere Bemühungen überschnitten.

Dein if macht total Sinn 

Sent on the go.

> Am 05.07.2016 um 05:50 schrieb lundefugl <notifications@github.com>:
> 
> In javafx7-mapview/src/main/java/slash/navigation/mapview/browser/JavaFX7WebViewMapView.java:
> 
> > @@ -84,6 +86,12 @@ protected WebView getWebView() {
> >      private WebView createWebView() {
> >          try {
> >              final WebView webView = new WebView();
> > +
> > +            final int scale = preferences.getInt(BROWSER_SCALE_FACTOR_PREFERENCE, 100);
> > +            if (scale != 100) {
> > +                setBrowserScale(webView, scale/100.);
> > +            }
> Das verstehe ich nicht ganz.
> Ich habe doch extra die If-Guard drin, damit die Funktion nur aufgerufen wird, wenn der Wert überschrieben wurde.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>